content/applications/marketing/events/revenues_report.rst

```diff
@@ -0,0 +1,161 @@
+===============
+Revenues report
+===============
+
+The Odoo *Events* application provides the ability to create custom reports, based on event-related
```

Comment:
```suggestion
The Odoo *Events* application allows users to create custom reports, based on event-related
```

Reply:
I've been instructed from ZST to avoid using the term "users" whenever possible. Hence, my wording here.
![Screenshot 2024-05-30 1221011](https://github.com/odoo/documentation/assets/73958186/effa68a3-1331-4f09-b09a-183a7d4ff2ca)